packages/react-components-lab/src/components/ResizeHandle/Handle.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import { Box } from '@material-ui/core';
+import {
+  DragHandle as DragHandleIcon,
+  ExpandLess as ExpandLessIcon,
+} from '@material-ui/icons';
+// eslint-disable-next-line import/no-cycle
```

Comment:
```suggestion
// eslint-disable-next-line import/no-cycle
```

Bit confused by these. I don't see a cycle, so are there actually errors here?

Reply:
Hmm not anymore, it was there because I was importing the type into the children, so it got triggered as a cycle on the project level.